orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -20,7 +20,7 @@ if [ $# -ne 5 ]; then
 fi
 
 CLOUD_PROVIDER=$(echo "$1" | tr '[:upper:]' '[:lower:]')
-CLOUD_LOCATION=$(echo "$2" | sed 's/\/\+$//g') # Clear trailng '/' (if present) for later path joining.
+BACKUP_LOCATION=$(echo "$2" | sed 's/[/]$//g') # Clear trailng '/' (if present) for later path joining.
```

Comment:
This has been renamed from CLOUD_LOCATION, but the rest of the script still expects it.